addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/json/AirQualityJsonCity.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal.json;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The {@link AirQualityJsonCity} is responsible for storing
+ * the "city" node from the waqi.org JSON response
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityJsonCity {
+
+    private String name;
+    private String url;
+    private List<Double> geo;
+
+    public AirQualityJsonCity() {
+
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public String getGeo() {
+        List<String> list = new ArrayList<String>();
+        for (int i = 0; i < geo.size(); i++) {
+            list.add(geo.get(i).toString());
+        }
+        return String.join(", ", list);
```

Comment:
join only exists in Java8, while your MANIFEST.MF states Java7 as "Bundle-RequiredExecutionEnvironment". You should adapt the MANIFEST.MF.